{"name": "test_add", "status": "failed", "statusDetails": {"message": "AssertionError: assert 15 == 16\n +  where 15 = <function Calc.add at 0x000001C503E66790>(1, 2, 12)\n +    where <function Calc.add at 0x000001C503E66790> = <src.a_pytest.b_sequence.Calc.Calc object at 0x000001C5041056A0>.add\n +      where <src.a_pytest.b_sequence.Calc.Calc object at 0x000001C5041056A0> = <src.a_pytest.b_sequence.test_case.TestClass object at 0x000001C504105670>.c", "trace": "self = <src.a_pytest.b_sequence.test_case.TestClass object at 0x000001C504105670>\n\n    def test_add(self):\n>       assert self.c.add(1, 2, 12) == 16\nE       assert 15 == 16\nE        +  where 15 = <function Calc.add at 0x000001C503E66790>(1, 2, 12)\nE        +    where <function Calc.add at 0x000001C503E66790> = <src.a_pytest.b_sequence.Calc.Calc object at 0x000001C5041056A0>.add\nE        +      where <src.a_pytest.b_sequence.Calc.Calc object at 0x000001C5041056A0> = <src.a_pytest.b_sequence.test_case.TestClass object at 0x000001C504105670>.c\n\nsrc\\a_pytest\\b_sequence\\test_case.py:19: AssertionError"}, "attachments": [{"name": "stdout", "source": "9dec4cc3-02a3-4952-825a-5a81cc821ba4-attachment.txt", "type": "text/plain"}], "start": 1578629214747, "stop": 1578629214747, "uuid": "917840e6-a74c-48f0-964a-5ca982acaff5", "historyId": "74f59845f0ee31bcdde039c10c0f1bdc", "testCaseId": "6d6ec2d922a6c434a19194f5b6d58a29", "fullName": "src.a_pytest.b_sequence.test_case.TestClass#test_add", "labels": [{"name": "parentSuite", "value": "src.a_pytest.b_sequence"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "GELOMEN-PC"}, {"name": "thread", "value": "4032-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.a_pytest.b_sequence.test_case"}]}